src/Compilers/CSharp/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,3 +1,9 @@
+*REMOVED*Microsoft.CodeAnalysis.CSharp.Syntax.ObjectCreationExpressionSyntax.ArgumentList.get -> Microsoft.CodeAnalysis.CSharp.Syntax.ArgumentListSyntax
```

Comment:
>ObjectCreationExpressionSyntax [](start = 46, length = 30)

Why are there changes here? I don't see any change to the syntax node definitions.

Reply:
Hmm, those are strange. I'll investigate.

Reply:
OK, figured it out, these are just re-ordered because I ran the sorting script to resolve merge conflicts.